packages/core/src/components/panel/panel.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, ComponentInterface, Event, EventEmitter, h } from "@stencil/core";
+
+/**
+ * @slot - The contents to be highlighted within the panel
+ * @slot heading - The heading (h2 - h6) with the title of the panel
+ */
+@Component({
+  tag: "dso-panel",
+  styleUrl: "panel.scss",
+  shadow: true,
+})
+export class Panel implements ComponentInterface {
+  /**
+   * Emitted when the user click the close button.
+   */
+  @Event()
+  dsoCloseClick!: EventEmitter<MouseEvent>;
```

Comment:
Graag eigen `PanelCloseEvent`, dan zijn we beter voorbereid op de toekomst.

Reply:
Done